tests/plugins/emoji/manual/font.html

```diff
@@ -0,0 +1,9 @@
+<textarea id="editor" cols="10" rows="10">
+	<p></p>
+</textarea>
+
+<script>
+	bender.tools.ignoreUnsupportedEnvironment( 'emoji' );
+
+	CKEDITOR.replace( 'editor' );
```

Comment:
Please also added as config `removeButtons: 'EmojiPanel'`, to have sure that nobody use toolbar icon.
_Using `:` at the beginning toolbar dropdown doesn't show proper emoji._

Reply:
TBO editor without toolbar looks broken. Why do you insist on hiding the editor toolbar? Test steps are clear, so it shouldn't be an issue.

Reply:
To guarantee that manual test checks exactly what it should. Having an icon in a toolbar provides an alternative path to insert an emoji in the editor. I see the possibility that tester will be in a rush or will have worse day and use toolbar icon instead of autocomplete panel. So if it doesn't cost us anything, then why we shouldn't use it? I think that having only one way to insert emoji is more clear and safer for this test.

Please also remember that toolbar will be filled with icons during the testing phase when the entire full-all build will be available for each manual test.

Reply:
After a short discussion, I agree we can remain it at is easy. There might be either editor without toolbar what might look as broken, or editor with at least one icon, which might suggest to use this icon instead of autocomplete panel.
However, reproduction steps are written in such way, that user should always use autocomplete panel not toolbar icon.